INSTALL.md

```diff
@@ -145,6 +174,39 @@ sudo ln -s /squashfs-root/AppRun /usr/bin/nvim
 nvim
 ```
 
+For arm64:
```

Comment:
same as above: pick one, either arm or x86. whichever is more common for linux users. we aren't going to repeat every variation of everything in excruciating detail.